python/lsst/drp/tasks/assemble_cell_coadd.py

```diff
@@ -114,6 +114,15 @@ class AssembleCellCoaddConfig(PipelineTaskConfig, pipelineConnections=AssembleCe
         "statistic. Passed to AccumulatorMeanStack.",
         default=True,
     )
+    mask_propagation_thresholds = DictField[str, float](
+        doc=(
+            "Threshold (in fractional weight) of rejection at which we "
+            "propagate a mask plane to the coadd; that is, we set the mask "
+            "bit on the coadd if the fraction the rejected frames "
+            "would have contributed exceeds this value."
+        ),
+        default={"SAT": 0.1, "INEXACT_PSF": 0.0},
```

Comment:
I'm confused as to why `INEXACT_PSF` is in this mapping at all - do we ever reject pixels because of `INEXACT_PSF`?  Is that effectively what's happening when we make a traditional cell-based coadd with no discontinuities?

I'd also have proposed a default for `SAT` over 0.5. Is this a value used in the current coaddition code?

Reply:
Ah, this is leftover from an earlier version of this ticket when I wanted to `INEXACT_PSF` mask bits sets. We've since moved away from that approach and will remove it. FWIW, unless mentioned here, all mask planes, whether defined or not, have a default threshold value of 0.0.

The value for `SAT` is based on what it is currently used in `assemble_coadd` but I don't have strong opinions on what the value should be.

Reply:
Ok, let's leave the SAT value the same as `assemble_coadd` for now and revisit it later, then.